VideoWeb/VideoWeb/ClientApp/src/sass/abstracts/_breakpoints.scss

```diff
@@ -0,0 +1,20 @@
+@import 'functions';
+
+$breakpoints: (
+  smallest: 0px,
+  extra-small : 480px,
+  small: 641px,
+  medium : 768px,
+  large: 1024px,
+);
+
+// 1024px
+$breakpoint-large: em(map-get($breakpoints, large));
+// 1023px
+$breakpoint-medium-max: em(map-get($breakpoints, large)-1px);
+// 768px
+$breakpoint-medium: em(map-get($breakpoints, medium));
+// 641px
+$breakpoint-small: em(map-get($breakpoints, small));
+// 480px
+$breakpoint-extra-small: em(map-get($breakpoints, extra-small));
```

Comment:
We should move away from using these or at least create them programmatically but leaving them for now to not interfere with places that use them.

Reply:
These were moved from variables.scss and changed to use the values in the map above.